docker-compose.yml

```
@@ -21,6 +23,13 @@ services:
       POSTGRES_INITDB_ARGS: --encoding=UTF-8
     volumes:
       - db-volume:/var/lib/postgresql/data
+    networks:
+      - todo_fastapi_default
 
 volumes:
   db-volume:
+
+networks:
+  todo_fastapi_default:
+    external:
+      name: todo_fastapi_default
```

docker network의 external 옵션은, 서로 다른 compose 끼리 통신할때 사용할 외부 네트워크를 명시하는 부분으로, 관리자가 사전에 'todo_fastapi_default'라는 네트워크를 따로 구성해두어야 사용 가능합니다.
이 프로젝트의 경우, 아래처럼만 해두어도 자동으로 internal todo_fastapi_default 네트워크가 생성되어 문제없이 컨테이너를 띄울 수 있습니다.
```suggestion
networks:
  todo_fastapi_default:
```

---

구두리뷰로 해결했습니다~!